Sources/Versioning/Commit.swift

```diff
@@ -13,7 +13,7 @@ public struct Commit {
     
     public init(string: String) throws {
         let components = string.split(separator: ": ")
```

Comment:
I think utilising `maxSplits` parameter would be a better implementation here:

```suggestion
        let components = string.split(separator: ": ", maxSplits: 1)
```

https://developer.apple.com/documentation/swift/string/split(separator:maxsplits:omittingemptysubsequences:)

Reply:
thats make more sense and didn't know we could do that. Does this mean we should change the guard statement back to `components.count == 2`?

Reply:
Yes I believe so!